tests/rptest/wasm/wasm_build_tool.py

```diff
@@ -53,7 +53,7 @@ def __del__(self):
         shutil.rmtree(self.work_dir)
```

Comment:
please fix the subject line (`rptest/wasm: The same filename for wasm script will hash to same id`) to state what the commit is doing.